Makefile

```diff
@@ -42,7 +42,7 @@ test-go: test-go-parallel test-go-serial
 
 .PHONY: test-go-parallel
 test-go-parallel:
-	go test ./... -race -timeout 30s
+	go test ./... -race -timeout 180s
```

Comment:
Do we want to increase the lifetime of all of these tests? In my `convertConfig` tests PR, I added a new command to the Makefile. You might check that out and see which you prefer.

Reply:
It shouldn't _really_ matter much, but I just approved https://github.com/0xProject/0x-mesh/pull/703 so let's go with that instead. I'll close this one.